src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -526,22 +664,34 @@ def executeMCPOnlyTestSteps(self, iteration_content, domain_proxy_objects):
     else:
       time.sleep(15)
 
+    protected_entity_records = iteration_content['protectedEntities']
+    # Creating FAD Object and Pull FAD records from SAS UUT
+    fad_test_harnesses_objects = []
```

Comment:
The test spec does not specify another FAD pull; please remove.

Reply:
Removed